src/Workspaces/CSharpTest/EmbeddedLanguages/RegularExpressions/CSharpRegexParserTests_BasicTests.cs

```diff
@@ -0,0 +1,15106 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Text.RegularExpressions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.EmbeddedLanguages.RegularExpressions
+{
+    public partial class CSharpRegexParserTests
+    {
+        [Fact]
+        public void TestEmpty()
+        {
+            Test(@"""""", @"<Tree>
+  <CompilationUnit>
+    <Sequence />
+    <EndOfFile />
+  </CompilationUnit>
+  <Captures>
+    <Capture Name=""0"" Span=""[0..0)"" />
+  </Captures>
+</Tree>", RegexOptions.None);
+        }
+
+        [Fact]
+        public void TestOneWhitespace_IgnorePatternWhitespace()
+        {
+            Test(@""" """, @"<Tree>
+  <CompilationUnit>
+    <Sequence />
+    <EndOfFile>
+      <Trivia>
+        <WhitespaceTrivia> </WhitespaceTrivia>
+      </Trivia>
+    </EndOfFile>
+  </CompilationUnit>
+  <Captures>
+    <Capture Name=""0"" Span=""[9..10)"" />
```

Comment:
Why is this span starting at 9, rather than 0?

Reply:
All the tests work by creating a statement of hte form:   `var v = <string literal>`
So everything is offset by 9 chars.  

This was actually itnentional (and i will doc) so that we don't accidently start emitting diagnostics as pos0 0.  We want to emit them at the right place in the doc itsel.f